src/snowflake/cli/api/project/schemas/updatable_model.py

```diff
@@ -160,7 +160,8 @@ def validator_skipping_templated_str(cls, value, handler, info: ValidationInfo):
 
         setattr(
             cls,
-            f"_field_validator_with_verbose_name_to_avoid_name_conflict_{field_name}",
+            # Unique name so that subclasses get a unique instance of this validator
+            f"_{cls.__module__}_{cls.__name__}_validate_{field_name}",
```

Comment:
nit - we can use `.` instead of `_` to separate module and name since __module__ will have . in it anyways.

Reply:
updated